configs/swintransformer/swin_tiny_ascend.yaml

```diff
@@ -0,0 +1,59 @@
+# system
+mode: 0
+distribute: True
+num_parallel_workers: 8
+val_while_train: True
+val_interval: 1
+
+# dataset
+dataset: "imagenet"
+data_dir: "path/to/imagenet"
```

Comment:
"path/to/imagenet" -> "/path/to/imagenet"

Reply:
fixed